thrift/sql/serializing.go

```diff
@@ -0,0 +1,59 @@
+package sql
+
+import (
+	"database/sql/driver"
+
+	"github.com/upfluence/errors"
+	"github.com/upfluence/thrift/lib/go/thrift"
+
+	"github.com/upfluence/pkg/thrift/serializer"
+	"github.com/upfluence/pkg/thrift/thriftutil"
+)
+
+var (
+	binaryThriftSerializer = serializer.NewTSerializer(
+		thriftutil.BinaryProtocolFactory,
+	)
+
+	binaryThriftDeserializer = serializer.NewTDeserializer(
+		thriftutil.BinaryProtocolFactory,
+	)
+)
+
+type TStructPtr[T any] interface {
+	thrift.TStruct
+	*T
+}
+type NullThrift[T any, PT TStructPtr[T]] struct {
```

Comment:
```suggestion
}

type NullThrift[T any, PT TStructPtr[T]] struct {
```
please remember that